README.md

```diff
@@ -27,6 +27,22 @@ Install the latest version of `confluent` to `/usr/local/bin`:
 
     brew install confluentinc/tap/cli
 
+#### YUM (RHEL and CentOS)
+
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `2.27` or above for `arm64`):
+
+    sudo rpm --import https://packages.confluent.io/confluent-cli/rpm/archive.key
+    sudo yum-config-manager --add-repo https://packages.confluent.io/confluent-cli/rpm/confluent-cli.repo
+    sudo yum clean all && sudo yum install confluent-cli
```

Comment:
Is `sudo yum clean all` required, or just a nice-to-have?

Reply:
Same question for `sudo apt update`

Reply:
Looks like neither is necessary. It's sometimes necessary for updates, though, so I'll add modify it a bit to mention that.

Reply:
Let's keep it if it's sometimes necessary, but delete it if it's never necessary... I'd rather give people one script to run that always works rather than discuss edge cases